core/controllers/classifier.py

```diff
@@ -79,7 +79,10 @@ def verify_signature(message, vm_id, received_signature):
     if secret is None:
         return False
 
-    generated_signature = generate_signature(secret, message)
+    if message is None:
```

Comment:
> @seanlip 
> Maybe something like status: success?

We are fetching job so what kind of status we should pass?
@pranavsid98 I didn't understand what do you mean by that?